Ghidra/Features/PDB/src/pdb/cpp/iterate.cpp

```diff
@@ -487,34 +474,38 @@ void iterateSections(IDiaEnumSectionContribs * pSecContribs) {
  * Accesses the program source code stored in the DIA data source.
  */
 void iterateInjectedSource(IDiaEnumInjectedSources * pInjectedSrcs) {
-	DWORD celt;
-	IDiaInjectedSource* pInjectedSrc;
+	DWORD celt = 0;
+    CComPtr<IDiaInjectedSource> pInjectedSrc;
 
 	while ( 1 ) {
-		HRESULT hr = pInjectedSrcs->Next( 1, &pInjectedSrc, &celt );
-		if (hr < 0) {
+		const HRESULT hr = pInjectedSrcs->Next( 1, &pInjectedSrc, &celt );
```

Comment:
lines 481-482: reduce to one line?  Eliminate hr and do if (FAILED(pInjected...)) ??